config.json

```diff
@@ -6,7 +6,7 @@
         }
     },
     "wit": {
-        "token": "CRPURQZRYEHGJGK6NSYRN2O67FOWSIKD",
+        "token": "",
```

Comment:
could we set this to a value to show it needs to be replaced (e.g. `replace-me`, or similar)?